### PR TITLE
triagebot: Add notification of 2024 issues

### PR DESCRIPTION
**Pull request by ehuss**
This adds a notification for 2024 issues to Zulip.

### CODE REVIEW

File: triagebot.toml [hunk 1]
```diff
@@ -424,7 +424,15 @@ message_on_reopen = "Issue #{number} has been reopened. Pinging @*T-types*."
 
 [notify-zulip."A-edition-2021"]
```
Comment (fmease) [inline triagebot.toml:425]: Should 2021 be removed or at least discriminated?

Comment (ehuss) [reply triagebot.toml:425]: I appreciate continuing to receive notifications, since they are still being filed.

Comment (ehuss) [reply triagebot.toml:425]: Oh, but I see what you're saying about noting the difference between them. Yea, I can add that, thanks!

**Review by fmease — APPROVED** [review]

**Pull request merged**